cloud/src/autogluon/cloud/predictor/cloud_predictor.py

```diff
@@ -221,7 +221,7 @@ def _setup_local_output_path(self, path):
         if path is None:
             utcnow = datetime.utcnow()
             timestamp = utcnow.strftime("%Y%m%d_%H%M%S")
-            path = f'AutogluonCloudPredictor/ag-{timestamp}{os.path.sep}'
```

Comment:
wondering how you capture this, is there any IDE hint?

Reply:
Wondering it too

Reply:
I was just using `os.path.sep` and going over these